.github/workflows/e2e.yml

```diff
@@ -9,7 +9,7 @@ env:
 
 jobs:
     test:
-        runs-on: macos-10.15
+        runs-on: macos-latest
```

Comment:
I vaguely remember this change being made because e2e tests were failing. I guess we'll see once they finish

Reply:
I think it was required before we moved to Xcode 12, but now that we are on Xcode 12 we are OK.